pkg/forwarder/forwarder.go

```diff
@@ -148,7 +147,7 @@ func (f *DefaultForwarder) retryTransactions(retryBefore time.Time) {
 	sort.Sort(byCreatedTime(f.retryQueue))
 
 	for _, t := range f.retryQueue {
-		if t.GetNextFlush().Before(retryBefore) {
+		if f.blockedList.willBeReady(t.GetTarget(), retryBefore) {
```

Comment:
Why not using `isBlock` since `retryBefore` is basically equal to `time.Now()` https://github.com/DataDog/datadog-agent/blob/308dbf50078d3f64e318951141ad8cdedec5ddfa/pkg/forwarder/forwarder.go#L186

I think we could simply remove `willBeReady`.

Reply:
Ah I didn't know that, thanks!